src/NUnitFramework/framework/Attributes/ExplicitAttribute.cs

```diff
@@ -62,8 +62,8 @@ public ExplicitAttribute(string reason)
         /// </summary>
         /// <param name="test">The test to modify</param>
         public void ApplyToTest(Test test)
-        {
-            if (test.RunState != RunState.NotRunnable)
+        {
+            if (test.RunState != RunState.NotRunnable && test.RunState != RunState.Ignored)
```

Comment:
The traditional NUnit behavior is that Ignore, being the "stronger" restriction, overrides Explicit. That makes sense if we view Ignore as being applied because there is a temporary problem.

There is some argument to be made about how it work, but we haven't called it out in Breaking Changes. We had some discussion in the context of issue #50 but decided to address it at a later stage.

My thought would be to first get this working correctly in the traditional interpretation.